client/core/trade.go

```diff
@@ -1164,11 +1197,12 @@ func (t *trackedTrade) processAudit(msgID uint64, audit *msgjson.Audit) error {
 
 	// Get the asset.AuditInfo from the ExchangeWallet. Handle network latency.
 	// The coin waiter will run once every recheckInterval until successful or
-	// until expiration after txWaitExpiration.
+	// until expiration after broadcast timeout. There is no reason to give up
+	// on the request sooner since the server will not ask again.
 	errChan := make(chan error, 1)
 	var auditInfo asset.AuditInfo
 	t.latencyQ.Wait(&wait.Waiter{
-		Expiration: time.Now().Add(txWaitExpiration),
+		Expiration: time.Now().Add(t.broadcastTimeout()),
```

Comment:
It's not clear to me that the broadcast timeout is the right value for latency expiration. They seem unrelated. Is there a good reason to use the broadcast timeout here?

Reply:
After broadcast timeout, the waiter might as well give up because the swap is going to be revoked.  Before broadcast timeout, why give up?

Reply:
I can imagine the waiter printing "tx still not found" messages periodically so the user can check out their node's connectivity, but otherwise I don't see a reason to go with a shorter expiration.